### ISSUE TITLE
Repetition of l.v. ties and damping

### ISSUE THREAD

**Issue by Dima-S-Jr** [musescore/MuseScore#25581]
### Your idea

I propose to implement a system that would make it possible to more specifically mark the end
of the l.v. by automatically repeating the l.v. ties over/after the subsequent barlines. 
Here is an example:
![Снимок экрана (730)](https://github.com/user-attachments/assets/d80b8318-3b43-40ec-8a95-83a2531dda6a)

I also propose to create a system for marking instructions on damping (in particular, this applies mainly to percussion). We are talking about a verbal or symbolic indication. In some cases, the player must be given an indication of exactly where the instrument to which the l.v. instruction was previously applied should be damped.
The instruction can be given both verbally (see the example above) and in the form of characters (caesura or a special character "Damp" (from the SMuFL). At the same time, if we are talking about verbal instructions, it is best to also use arrows so that the player understands exactly where the instrument should be damped, and it would also be nice if the word "damp" was automatically centered above/below the arrow (or it was "slightly" off-center if the place does not allow it):
![Снимок экрана (731)](https://github.com/user-attachments/assets/690a7150-bcb2-4292-8930-803fbb81281c)
In this case, the "damp" instruction should have the property not just as "Staff text", but a special indication affecting playback (i.e. damping the sound in the appropriate place). Also, such instructions should be available for translation into app.transifex.com

Take a look at how instrument damping can be indicated using the symbol:
![Снимок экрана (732)](https://github.com/user-attachments/assets/9ffdd6a3-5cc5-4254-865e-aebeb7c47eaf)

 For the benefit of the score-reader, open ties may indicate the continuing resonances of sustained notes as well, either after each note (or chord), or across each barline. It would be nice to have a score-style setting for this designation so that the ties are automatically displayed when a sound-sustaining pedal is added. 
![Снимок экрана (733)](https://github.com/user-attachments/assets/a457a7d9-14fa-43b9-b61c-99938279fe6d)

### Problem to be solved

Currently, there is no function in MuseScore Studio that allows you to dampen the sound in any particular place. A priori, it is intended that the resonance will be until it subsides in a natural way. This corresponds to the term "_l.v. al niente_". This is about the sound.
With regard to engraving: it is practically possible to achieve such a notation, but it will be nothing more than a workaround with a lot of manual adjustments. I'm raising the issue of automating the process. But I think that it will be possible to approach this issue after #19753 (as well as #21685) is resolved.

And yes, I almost forgot. Why do we need repeatable l.v. ties? Well, first of all, it serves as a reminder that l.v. is still valid and valid up to a certain point. This is visually useful for the performer.

### Additional context

Repeatable l.v. ties can be used not only for percussion. Take a look, for example, how it can be designated for a chord in grand staff:
![01_-_Full_score_-_Les_soirs_illumines_par_lardeur_du_charbon_-_002](https://github.com/user-attachments/assets/6e32dd32-6779-4791-a737-1d8ec753e543)
It would be great if this was also implemented.

### Checklist

- [X] This request follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting issues
- [X] I have verified that this feature request has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar requests

**Comment by rettinghaus** [comment]
A printed "real world" example

![grafik](https://github.com/user-attachments/assets/09a177ce-fae5-4de4-9519-bef4158cd8fb)
